src/shared/components/lazyMobXTable/LazyMobXTable.tsx

```diff
@@ -378,7 +378,29 @@ export class LazyMobXTableStore<T> {
                 }
             );
         });
-        return tableDownloadData;
+
```

Comment:
I don't like that we process all the columns and _then_ filter out columns without the download function. Futhermore, you're just filtering out empty columns, which could theoretically be legit data. 

Can you spend some time seeing if you can instead filter out columns before we create tableDownloadData?

Reply:
That's true- I just realized it's inefficient to filter at the very end (and a specified download function could return the empty string). I just saw that the header and an empty string for the download object rows are displayed for columns without specified download functions.